background/setBackground.js

```diff
@@ -0,0 +1,5 @@
+const setBackground = (weatherIcon) => {
```

Comment:
You could actually pass the weatherCode here directly, instead of passing whole weatherIcon. You're just using one property of that!

Reply:
Done